utils/bash/Makefile

```diff
@@ -40,22 +39,22 @@ define Package/bash/description
   incorporates useful features from the Korn and C shells (ksh and csh).
 endef
 
-
-define Build/Configure
-	$(call Build/Configure/Default, \
-		--without-bash-malloc \
-                --bindir=/bin \
-	)
-endef
-
-
-define Build/Compile
-	$(MAKE) -C $(PKG_BUILD_DIR)/builtins LDFLAGS_FOR_BUILD= mkbuiltins
-	$(MAKE) -C $(PKG_BUILD_DIR) \
-		DESTDIR="$(PKG_INSTALL_DIR)" \
-		SHELL="/bin/bash" \
-		all install
-endef
+CONFIGURE_VARS += \
+	ac_cv_rl_prefix="$(STAGING_DIR)/usr" \
+	ac_cv_rl_version="7.0" \
+	bash_cv_getcwd_malloc=yes \
+	bash_cv_job_control_missing=present \
+	bash_cv_sys_named_pipes=present \
+	bash_cv_func_sigsetjmp=present \
+	bash_cv_printf_a_format=yes \
+	bash_cv_getenv_redef=yes
```

Comment:
Please add a note explaining why these presets are needed.

Reply:
Something like this?

```
bash_cv_sys_named_pipes --> http://www.linuxfromscratch.org/clfs/view/svn/mips64/temp-system/bash.html
bash_cv_func_sigsetjmp --> https://git.openwrt.org/?p=openwrt/openwrt.git;a=blob;f=package/libs/readline/Makefile
bash_cv_printf_a_format --> Needed for job control (stripped down in later versions)
https://git.busybox.net/buildroot/commit/package/bash/bash.mk?id=6f6893c633ef6d31c467a4154088b2962c93fd77
bash_cv_getenv_redef --> Fix for non uclibc toolchains
https://git.busybox.net/buildroot/commit/package/bash?id=9623ff651f83a350819719f10705552da9793205
```

Reply:
Text that can help other people including current maintainer, future maintainers, reviewers understand why these additional tweaks are necessary and useful is good.  Otherwise these things pile up over time and turn into dark magic leaving people searching around whether they can drop these customisations on a specific version bump.

Please write something that conclude what you already take time and efforts to know about these things so that it can also help others.  Thank you